.github/workflows/ci.yml

```diff
@@ -86,4 +86,24 @@ jobs:
         with:
           go-version-file: go.mod
           cache: false
+
+      - uses: elastic/apm-pipeline-library/.github/actions/docker-login@current
+        with:
+          registry: docker.elastic.co
+          secret: secret/observability-team/ci/docker-registry/prod
+          url: ${{ secrets.VAULT_ADDR }}
+          roleId: ${{ secrets.VAULT_ROLE_ID }}
+          secretId: ${{ secrets.VAULT_SECRET_ID }}
+        if: |
+          github.event_name != 'pull_request'
+          || github.event_name == 'pull_request' && github.event.pull_request.head.repo.fork == false
+
       - run: make package-snapshot
+        env:
+          GH_TOKEN: ${{ github.token }}
```

Comment:
Remove some warnings in the github logs, regarding missing variable when running gh in the GitHub runner